src/main/java/com/rocana/lucene/codec/v1/RocanaBlockTreeTermsReader.java

```diff
@@ -44,7 +45,28 @@
 import org.apache.lucene.util.fst.ByteSequenceOutputs;
 import org.apache.lucene.util.fst.Outputs;
 
-/** A block-based terms index and dictionary that assigns
+/**
+ * Fork of Lucene's {@link com.rocana.lucene.codec.v1.RocanaBlockTreeTermsReader}.
+ *
+ * Why we forked:
+ *   - We can't extend because it has final fields that must be set in the constructor,
+ *     and the constructor is the code we need to modify.
+ *
+ * What changed in the fork?
+ *   - Commented out the constructor's call to {@link CodecUtil#checksumEntireFile(IndexInput)}
+ *     for performance.
```

Comment:
Maybe add: "for performance improvements and on HDFS this is a redundant check to the checksumming of blocks already done by HDFS".